CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
```

Comment:
Add an intro similar to this (generated with ChatGPT). This still needs adjustment:

```
# Contributing to EvmosJS

Welcome to the EvmosJS repository! We appreciate your interest in contributing to the project.

Before submitting a contribution, please take a moment to review this document to ensure that your contribution follows our guidelines and to help make the contribution process as smooth as possible.

## Getting Started

If you're new to contributing to open source projects, you may find it helpful to read [GitHub's Guide to Contributing to Open Source](https://opensource.guide/how-to-contribute/).

To contribute to EvmosJS, follow these steps:

1. Fork the repository.
2. Clone the forked repository onto your local machine.
3. Create a new branch for your changes. It is recommended that you name the branch according to the issue you are addressing (e.g. issue-123).
4. Make your changes to the code.
5. Ensure that your changes pass the existing tests by running npm test.
6. Commit your changes with a descriptive commit message.
7. Push your changes to your forked repository.
8. Open a pull request to the original repository, explaining the changes you've made and referencing the issue number(s) the changes relate to.

## Guidelines

Here are some guidelines to keep in mind when contributing to EvmosJS:

### Issues and Pull Requests

Before starting work on a new feature or bug fix, check the [issues](https://github.com/evmos/evmosjs/issues) to see if it has already been reported. If it hasn't, please create a new issue before starting work.

When submitting a pull request, please ensure that:

* Your changes address the issue you're referencing.
* Your code follows our coding standards.
* Your commit messages are descriptive and follow our guidelines.

### Code Standards

When submitting code changes, please ensure that your code follows our coding standards:

* Use two spaces for indentation.
* Use single quotes for strings.
* Use camelCase for variable and function names.
* Use const and let instead of var.
* Use semicolons at the end of statements.

### Testing

We use Jest for testing. Before submitting a pull request, please ensure that your changes pass the existing tests by running npm test.

### Documentation

If you're making changes to the public API, please update the documentation in the README.md file accordingly.

## Conclusion

Thank you for contributing to EvmosJS! Your contributions help us make the project better. If you have any questions or suggestions, please don't hesitate to create a new issue.
```
```